scripts/render_DESeq2_report.R

```diff
@@ -86,8 +86,13 @@ if (is.na(params$group_facet)) {
              c(params$exclude_groups, skip_extra)) %>%
     pull(params$group_facet) %>% 
     unique()
+  facets <- facets[grep(pattern = "DMSO", x = facets, invert = T)]
   message(paste0("Making multiple reports based on ",
                  params$group_facet ,"..."))
+  print(facets)
+  # Initialize a summary file
+  make_table <- paste0('touch ', file.path(params$projectdir, "DEG_output/DEG_summary.txt"))
```

Comment:
I think the file name was `paste0(NORM_TYPE, "-DEG_summary.txt"))`. But this shouldn't be necessary anyway?

Reply:
👍